css/styles.css

```diff
@@ -268,3 +268,71 @@ textarea {
   resize: vertical;
   height: 170px;
 }
+
+@media (min-width: 500px) {
+  body {
+    background-color: #172b4d;
+  }
+
+  .app-wrapper {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .welcome-line {
+    background-image: url("/assets/shapes/background-shapes/desktop/desktop-header-shapes.svg");
+  }
+
+  .portfolio-cards {
+    margin: 142px;
+    gap: 142px;
+  }
```

Comment:
- [ ] The contrast looks great. However, it's one of the requirements to replicate the Figma design. Please make sure you use the right background color, color, font.

### Your implementation
![image](https://user-images.githubusercontent.com/59416136/154526070-8d6b56b2-d42e-4644-8bee-284db4cb2589.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/154526237-f43461e7-7a8b-438e-a406-a37b2147e0b6.png)